public_html/wp-content/mu-plugins/blocks/assets/src/sponsors/block-controls.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies.
+ */
+import Placeholder from 'react-select/src/components/Placeholder';
```

Comment:
There is a Placeholder component in Core's `wp.components`. Did you mean to use the one from `react-select`?